src/lib/utils/mem_ops.h

```diff
@@ -8,6 +8,7 @@
 #ifndef BOTAN_MEMORY_OPS_H_
 #define BOTAN_MEMORY_OPS_H_
 
+#include <botan/allocator.h>
```

Comment:
Do we need to mention in a comment that this include is here for backward compatibility? Maybe even with a link to the discussion on GitHub. There's a non-zero chance that we'll run include-what-you-use or similar and will just drop that.